src/spatial/models/PMcamsDataModel.py

```
@@ -0,0 +1,235 @@
+# models/cams_data_controller.py
+import os
+import json
+from datetime import datetime, timedelta
+import numpy as np
+import pandas as pd
+from flask import jsonify, send_file, Response
+from redis import Redis
+from sqlalchemy import create_engine, text
+import rasterio
+from rasterio.transform import from_origin
+import geopandas as gpd
```

Ruff (0.11.9)

`geopandas` imported but unused

Remove unused import: `geopandas`

(F401)